content/influxdb/v2.0/reference/flux/stdlib/influxdb-v1/_index.md

```diff
@@ -15,6 +15,11 @@ weight: 202
 influxdb/v2.0/tags: [functions, influxdb-v1, package]
 ---
 
+{{% note %}}
+Functions from the `v1` package have moved to [`schema`](/influxdb/v2.0/reference/flux/stdlib/schema).
```

Comment:
Not all the functions moved, just most. Specifically the json and databases functions remain in the v1 package.